easybuild/easyconfigs/a/ABINIT/ABINIT-8.10.2-intel-2018b.eb

```diff
@@ -0,0 +1,47 @@
+easyblock = 'ConfigureMake'
+
+name = 'ABINIT'
+version = '8.10.2'
+
+homepage = 'https://www.abinit.org/'
+description = """ABINIT is a package whose main program allows one to find the total energy,
+ charge density and  electronic structure of systems made of electrons and nuclei (molecules
+ and periodic solids) within Density Functional  Theory (DFT), using pseudopotentials and a
+ planewave or wavelet basis."""
+
+toolchain = {'name': 'intel', 'version': '2018b'}
+toolchainopts = {'usempi': True, 'pic': True}
+
+source_urls = ['https://www.abinit.org/sites/default/files/packages/']
+sources = [SOURCELOWER_TAR_GZ]
+checksums = ['4ee2e0329497bf16a9b2719fe0536cc50c5d5a07c65e18edaf15ba02251cbb73']
+
+dependencies = [
+    ('libxc', '3.0.1'),
+    ('netCDF', '4.6.1'),
+    ('netCDF-Fortran', '4.4.4'),
+    ('HDF5', '1.10.2'),
+]
+
+# ensure mpi and intel toolchain
+configopts = '--enable-mpi '
+configopts += '--enable-gw-dpc '
+configopts += '--with-linalg-flavor=mkl --with-fft-flavor=dfti --with-fft-libs="$LIBFFT" '
```

Comment:
@jmbeuken is there any performance benefit with `dfti` vs. `fftw3-mkl`?

Reply:
These two options are equivalent when the code is linked with MKL as  fftw3-mkl is just wrapping  the native MKL FFT library (DFTI) with an API that is compatible with FFTW3.
fft-flavor DFTI calls the MKL library directly so it's supposed to be slightly faster.

Note, however, that fftw3-mkl should not be used if the threaded FFT algorithm is wanted as  
the wrappers are not thread-safe.